cfgov/unprocessed/js/modules/util/behavior.js

```diff
@@ -0,0 +1,75 @@
+/* ==========================================================================
+   Dom Behaviors
+   ========================================================================== */
+
+'use strict';
+
+// Required modules.
+var standardType = require( './standard-type' );
+
+/**
+ * @param {string} behaviorSelector
+ *  Behavior type used to find the element within the dom.
+ * @param {HTMLNode} baseElement Containing element for the behavior element.
+ * @returns {HTMLNodeList} if it exists in the dom, null otherwise.
+ */
+function _findElements( behaviorSelector, baseElement ) {
+  baseElement = baseElement || document;
+  return baseElement.querySelectorAll( behaviorSelector );
+}
+
+/**
+ * @param {( string|HTMLNode|HTMLNodeList )} behaviorElement
```

Comment:
I ended up leaving this as-is to mimc https://api.jquery.com/jQuery/ ( kind of ironic ).